guardian/bonobo
# Show last used times for keys

## What does this change?

- Updates Bonobo to display "days since this key was last used" data that will be input to the `keys` table by KongKeyMon

**NOTE** Requires https://github.com/guardian/kongkeymon/pull/12 to push data into the table. However, it should be able to run on non-updated tables without a problem, simply displaying "Not Detected" for every key

![Screenshot 2022-08-22 at 17 19 34](https://user-images.githubusercontent.com/12482441/185970040-5c474e40-048a-45cd-afb1-72362a54f492.png)


## How to test

- View the keys table and find one of yours that has not been used
- Click it to get to the user and notice the same values in the 'Keys for this user' table
- Make a couple of requests via Open Platform code
- Give it a couple of seconds for the data to come through
- Run the KongKeyMon process that extracts the data (see seperate forthcoming PR)
- Refresh Bonobo and see the new data

## How can we measure success?

Easy viewing of active/inactive keys

## Have we considered potential risks?

n/a
